frontend/src/components/CoursePreview/helper.ts

```diff
@@ -0,0 +1,5 @@
+export const timeHandler = (minutes: number): string  => {
+    const hours: number = minutes / 60;
+    if (hours < 1) return ('' + minutes + ' minutes');
+    return '' + hours + ' hours' + minutes%60 + 'minutes';
```

Comment:
Rewrite it to:
`${hours} hours ${minutes % 60} minutes`

Reply:
Done. You are right, curly braces looks much better. Bad java habit